tools/install.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
 #
-# This will:
-#   1. Set up the LORIS DB schema
-#   2. Log the installation in the logs directory
-# This will only install the database components and LORIS config file.
+# This script will:
+#   1. verify composer is installed to manage dependencies
+#   2. Set up the directory structure with appropriate permissions
+#   3. Log the installation in the logs directory
```

Comment:
To do: change "the installation" to "this script's output"

Reply:
- [x] done : see my [commit 475ca8](https://github.com/aces/Loris/pull/5205/commits/475ca805073aba4296b34ac0698d3e16e4a5d3fa)